.github/workflows/dockerhub-released-image.yml

```diff
@@ -26,6 +26,10 @@ jobs:
           # 0 indicates all history for all branches and tags.
           # for `git describe --tags` in Makefile.
           fetch-depth: 0
+      - name: install Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.17.x
```

Comment:
line 45 `run: make mp-image-${{ matrix.target }}` need to be fixed.

Reply:
What's wrong with it? Sorry, i cannot get it...

Reply:
In this yml line 45 we will build multi-arch images using `make mp-image-xxx` cmd, but this PR have deleted make mp-image-xxx logic in Makefile, we need to fix this
```
- name: build and publish images
        env: 
          VERSION: ${{ github.ref_name }}
        run: make mp-image-${{ matrix.target }}
```

Reply:
`make mp-image-xxx` are merged here, this rule is still work.
```
# Build and push multi-platform image to DockerHub
#
# Example
#   make multi-platform-images
#   make mp-image-karmada-aggregated-apiserver
MP_TARGET=$(addprefix mp-image-, $(TARGETS))
.PHONY: $(MP_TARGET)
$(MP_TARGET):
	set -e;\
	target=$$(echo $(subst mp-image-,,$@));\
	make $$target GOOS=linux GOARCH=amd64;\
	make $$target GOOS=linux GOARCH=arm64;\
	VERSION=$(VERSION) REGISTRY=$(REGISTRY) \
		OUTPUT_TYPE=registry \
		BUILD_PLATFORMS=linux/amd64,linux/arm64 \
		hack/docker.sh $$target
```

https://github.com/ikaven1024/karmada/blob/46ce68f606a182e1105abb149deb7e8be7598eb8/Makefile#L75-L90

Reply:
I merged `make xxx`, `make image-xxx`, `make mp-image-xxx`. It reduce the code, and easy to maintain. When new component is added, just append it at 
```
TARGETS := karmada-aggregated-apiserver \
			karmada-controller-manager \
			karmada-scheduler \
			karmada-descheduler \
			karmada-webhook \
			karmada-agent \
			karmada-scheduler-estimator \
			karmada-interpreter-webhook-example
```

Reply:
oh, got it

Reply:
by test, these below cmds are ok, make one components
```
make karmada-webhook
make image-karmada-webhook
make mp-image-karmada-webhook
```

and for all components, we now support
```
make all
make images
```
but not support make all mp images.

Reply:
/lgtm

Reply:
> but not support make all mp images.

`make multi-platform-images` did it. This rule is added before, I didn't change its name.

Reply:
oh, sry,